functests/1_performance/cpu_management.go

```diff
@@ -18,6 +19,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
+
```

Comment:
Is the empty line created by an automated code formatter?

Reply:
hm, Goland did it itself..

Reply:
ok